assets/src/edit-story/app/prepublish/guidance/media.js

```diff
@@ -134,3 +135,26 @@ export function videoElementLength(element) {
   }
   return undefined;
 }
+
+/**
+ * Check a if a video element's been optimized.
+ * If is not optimized, return guidance. Otherwise return undefined.
+ *
+ * @param {Element} element The element being checked
+ * @return {Guidance|undefined} The guidance object for consumption
+ */
+export function videoElementOptimized(element) {
```

Comment:
i think we should also check against 3rd party videos from `coverr` since we can't re-upload those? 

We could check for `resource.id` to start with `media/coverr` or we could check `resource.src` to start with `https://storage.coverr.co/videos/` to eliminate those from results.  Or if there's a more basic way to track this that I'm missing that's awesome.